vm/src/hint_processor/builtin_hint_processor/blake2s_utils.rs

```diff
@@ -226,31 +217,27 @@ pub fn blake2s_add_uint256_bigend(
     let data_ptr = get_ptr_from_var_name("data", vm, ids_data, ap_tracking)?;
     let low_addr = get_relocatable_from_var_name("low", vm, ids_data, ap_tracking)?;
     let high_addr = get_relocatable_from_var_name("high", vm, ids_data, ap_tracking)?;
-    let low = vm.get_integer(low_addr)?.into_owned();
-    let high = vm.get_integer(high_addr)?.into_owned();
+    let mut low = vm.get_integer(low_addr)?.into_owned();
+    let mut high = vm.get_integer(high_addr)?.into_owned();
     //Main logic
-    //Declare constant
-    const MASK: u32 = u32::MAX;
-    const B: u32 = 32;
-    //Convert MASK to felt
-    let mask = Felt252::new(MASK);
+    let b = &Felt252::TWO.pow(32_u8).try_into().unwrap();
```

Comment:
can we make this a constant?

Reply:
I'll try to do a quick change to group our `lazy_static` constants, yes.